test/ut3_user/expectations/test_expectation_anydata.pks

```diff
@@ -203,5 +203,29 @@ create or replace package test_expectation_anydata is
  
   --%test ( Empty Array not equal array with space )
   procedure arr_empty_nqua_arr_e_unord;  
+  
+  --%test ( Failure of comparing nested objects )
+  procedure failure_nesting_objects;
+ 
+  --%test ( Failure of comparing double nested objects ) 
+  procedure failure_double_nested_objects;
+  
+  --%test ( Success of comparing nested objects )
+  procedure success_nesting_objects;
+  
+  --%test ( Success of comparing double nested objects ) 
+  procedure success_double_nested_objects;  
+ 
+  --%test ( Failure of comparing nested object list )
+  procedure failure_nested_object_list;
+ 
+  --%test ( Success of comparing nested object list )
+  procedure success_nested_object_list;
+  
+  --%test(Nested VARRAYS with same data)
+  procedure nested_varray_same_data;
+  
+  --%test(Nested VARRAYS with different data)
+  procedure nested_varray_diff_data;  
```

Comment:
Can you add a test to cover for regression of #1098 (on nested element being null)?

Reply:
Thats test replicated using our types and regression created.